docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -136,7 +136,20 @@ A template variable of the type `Constant` will automatically be hidden in the d
 
 The import process enables you to change the name of the dashboard, pick the data source you want the dashboard to use, and specify any metric prefixes (if the dashboard uses any).
 
-### Discover dashboards on Grafana.com
+### Set up generative AI features for dashboards
+
+{{< docs/public-preview product="dashgpt" featureFlag="dashgpt" >}}
+
+You can use generative AI to help you with the following tasks:
+
+- **Generate panel and dashboard titles and descriptions**: Generate a title and description for your panel or dashboard based on the data you’ve added to it. This is useful when you want to quickly visualize your data and don’t want to spend time coming up with a title or description.
+- **Generate dashboard save changes summary**: Generate a summary of the changes you’ve made to a dashboard when you save it. This is great for easily tracking the history of a dashboard.
+
+To access these features, enable the `dashgpt` feature toggle. Then install and configure Grafana’s Large Language Model (LLM) app plugin. For more information, refer to the [Grafana LLM plugin documentation][].
+
+When enabled, the **✨ Auto generate** option displays next to the **Title** and **Description** fields in your panels and dashboards, or when you press the **Save** button.
```

Comment:
What is ✨ right before the word Auto?

Reply:
it's literally a star emoji that shows up in the UI